### ISSUE TITLE
Please change the name and icon to a more professional one

### ISSUE THREAD

**Issue by andrew712-1** [SchildiChat/SchildiChat-android#84]
Please change the icon, the current one is too childish. 

Please change the name of the app, it's too long, and hard to pronounce especially for russians. We need a shorter and easier to pronounce (internationally) name.

I love your app, but I can't recommend it to anyone because of these two issues...

**Comment by andrew712-1** [comment]
I can help with ideas and also designing an icon, if you wish...

**Comment by SpiritCroc** [comment]
A name change to a more international one would need to be somehow related to the current name, I don't think a full re-brand is worth the effort. If you have a really good idea, I might consider it, but probably won't happen anytime soon.

I see that the name might look hard to pronounce, but I don't think it actually is hard to pronounce if you know how to. For the same pronunciation in English, you might write something like "Shilldy-Chat", I guess.

"Too long" - well, it's three syllables, so same as "Element", but way easier to search for in the Internet. I don't see the need to make it any shorter, while risking being less searchable.

We're open to a better version of the icon, if it maintains the original traits.

**Comment by andrew712-1** [comment]
What does the app stand for? What are its main traits? What is it trying to be?
How is it different from other similar chats? Is it pro-free-speech or pro-anonymity, or ..? What is your mission?
What does Schildi mean? What does the turtle from the logo mean?

I need to know this in order to come up with some suggestions. Also, are you ok with dropping the "chat" word from the app name? None of the mainstream (or popular) chatting apps do that (Element, Riot, Signal, Telegram, etc..).

**Comment by SpiritCroc** [comment]
"Schildi" comes from "Schildkröte", which is turtle in German. We don't have any "mission" or philosophical "what the app stands for", we just develop a chat client that we want to use for ourselves, and share it with the community. Accordingly, you might have noticed it's not a priority for us to have a "professional" name, or how you call it.

"chat" is not required, but it is good if the app name says something about what the app does.

If you want an explanation of what the logo stands for, here is some that I came up with (as a joke) after everything was designed already:
- Turtle shell: stands for security
- Turtle feet: stand for stability
- Posthorn: stands for connectivity

**Comment by andrew712-1** [comment]
Turchat (**Tur**tle + **chat**)
Chatur (**Chat** + t**ur**tle)
T-chat (Turtle chat)

The logo could look something like this:

![kkk](https://user-images.githubusercontent.com/89605550/132997657-16207afd-5bc2-436b-ae52-c67bd2c56d11.png)

**Comment by SpiritCroc** [comment]
Thanks for your input. I'm not persuaded though, I still prefer the old icon and name.

**Comment by HerrFrutti** [comment]
Just installed the app and I like it very much! First thing I thaught, this logo is a little off.
From what I read, you don't want to looseyour identity.
What do you think from this **sketch**, I'd be happy to work something out with you :)
![sketch1637608210896](https://user-images.githubusercontent.com/6975702/142921489-43c0ce1c-3869-4f66-916f-2094e80ae1e2.png)

**Comment by SpiritCroc** [comment]
@HerrFrutti
Great logo idea, we like it!

The only thing that bothers me right now is that the shape of the post horn reminds me slightly too much of the "Deutsche Post" logo - maybe one with a little less bending would do, similar to this one? https://upload.wikimedia.org/wikipedia/commons/thumb/f/f4/Berlin._Spandau._Citadel_057.JPG/1280px-Berlin._Spandau._Citadel_057.JPG

Do you have experience with designing icons and/or vector graphics?

**Comment by HerrFrutti** [comment]
Yeah I taught of Deutsche Post when reading Posthorn :D
Me myself not, but my girlfriend (Mediengestalterin). If I can tell her what we want she can do the magic.
That sketch is from me, obviously.

I'll talk to her and update you on a more developed version of my sketch.

**Comment by squid-f** [comment]
Hi. The current logo doesn't really bother me. That being said, if a new one is looked at, I would recommend to use something more "universal" than the horn, on the turtle back.
My 2 cents. :)

**Comment by andrew712-1** [comment]
Great! Thanks guys for your understanding! 

I hope we're being understood, we love your app but we can't recommend it to our friends because it gives a first impression of being:  **weird, childish and with a hard to pronounce name.**  But when you get past this hurdle you discover a very great app!

It'd be amazing if you changed the name to something more easy to pronounce for as many people from different countries as possible!

**Honestly, thank deeply for your understanding!**

**Comment by SpiritCroc** [comment]
> Hi. The current logo doesn't really bother me. That being said, if a new one is looked at, I would recommend to use something more "universal" than the horn, on the turtle back. My 2 cents. :)

I'm open for suggestions

**Comment by andrew712-1** [comment]
> > Hi. The current logo doesn't really bother me. That being said, if a new one is looked at, I would recommend to use something more "universal" than the horn, on the turtle back. My 2 cents. :)
> 
> I'm open for suggestions

Maybe a **lockpad** which symbolizes security and privacy.

**Comment by SpiritCroc** [comment]
> Maybe a **lockpad** which symbolizes security and privacy.

Then we would have no indicator of the messenger aspect.

**Comment by HerrFrutti** [comment]
Yeah, I'm not having that much time right now, but I was working on an updated logo. I was thinking of a big shield, that has a divided logo on it. One site is the turtle and on the other the horn.
But I'm not finding the time right now to create it... sorry

**Comment by squid-f** [comment]
> > Hi. The current logo doesn't really bother me. That being said, if a new one is looked at, I would recommend to use something more "universal" than the horn, on the turtle back. My 2 cents. :)
> 
> I'm open for suggestions

Hi
See below. The link between the round magnets could draw a horn, for instance. Number of magnets could be adjusted. 
[schidli-icon.png](https://postimg.cc/XXZ4HJzn)
If you like the principle, I could try to make a proposal (caution: I am not at all a talented web designer :) ) or someone better than me can take over.

**Comment by SpiritCroc** [comment]
> > > Hi. The current logo doesn't really bother me. That being said, if a new one is looked at, I would recommend to use something more "universal" than the horn, on the turtle back. My 2 cents. :)
> > 
> > 
> > I'm open for suggestions
> 
> Hi See below. The link between the round magnets could draw a horn, for instance. Number of magnets could be adjusted. [schidli-icon.png](https://postimg.cc/XXZ4HJzn) If you like the principle, I could try to make a proposal (caution: I am not at all a talented web designer :) ) or someone better than me can take over.
> 
> EDIT: to make sure my idea is clear, it is to put onto the turtle shell sketch proposed by HerrFrutti

That looks like WAY too much detail for an icon.

**Comment by andrew712-1** [comment]
On the back of the turtle we could have something that symbolises "Chat" like:

1:

<img src="https://user-images.githubusercontent.com/89605550/145689102-2329e9ef-041c-4436-859c-272e4d6e865c.png" width="100" />

2: 
<img src="https://user-images.githubusercontent.com/89605550/145689111-522a1158-eda8-4b2c-982e-3417f26e9006.png" width="100" />

3:

<img src="https://user-images.githubusercontent.com/89605550/145689332-8f11ae01-fda8-4cf7-9d10-857793f6994a.png" width="100" />

**Comment by HerrFrutti** [comment]
just another quick sketch:
![sketch1639252383961](https://user-images.githubusercontent.com/6975702/145689829-edabb95c-9ddf-4fd8-a103-b1a93e2e6f13.png)

**Comment by SpiritCroc** [comment]
> On the back of the turtle we could have something that symbolises "Chat" like:
> 
> 1:
> 
> <img alt="" width="100" src="https://user-images.githubusercontent.com/89605550/145689102-2329e9ef-041c-4436-859c-272e4d6e865c.png">
> 
> 2: <img alt="" width="100" src="https://user-images.githubusercontent.com/89605550/145689111-522a1158-eda8-4b2c-982e-3417f26e9006.png">
> 
> 3:
> 
> <img alt="" width="100" src="https://user-images.githubusercontent.com/89605550/145689332-8f11ae01-fda8-4cf7-9d10-857793f6994a.png">

Doesn't really persuade me, and also seems like too much detail. These could be icons on their own, but if you put it into another icon, it just gets too small to tell what it is. Keep in mind that an app icon is just a small icon on the screen in many scenarios. Furthermore, 1. takes too much thinking what this actually is, and (2. and) 3. would be kind of copying other apps, and also kind of boring, compared to the horn, if you ask me.


> just another quick sketch: ![sketch1639252383961](https://user-images.githubusercontent.com/6975702/145689829-edabb95c-9ddf-4fd8-a103-b1a93e2e6f13.png)

Hm, doesn't persuade me. Your first sketch looked perfect to me, not sure if you're doing new sketches because I said I'm "open for suggestions" for other things to put on the turtles back than a posthorn, or just to have more alternatives to choose from.

**Comment by HerrFrutti** [comment]
good to know, if I find the time, I'll look deeper into my first sketch.

I just tried different things because my first idea feels complicated to get right. As you said, an icon is small and to much detail is bad. Combining three objects may be too much. But I think we should give it a try and any one can be inspired by the first sketch.

**Comment by squid-f** [comment]
Hi
I am not good at all at drawing. Here below, I tried to illustrate the idea I had to play with symbols and lines to replace the post-horn. 
![schildi1](https://user-images.githubusercontent.com/11743181/146949342-7937650d-d271-463a-acf3-5669443986eb.png)

**Comment by SpiritCroc** [comment]
> Hi I am not good at all at drawing. Here below, I tried to illustrate the idea I had to play with symbols and lines to replace the post-horn.

Still looks like a post horn, I thought you wanted to put something else there :thinking:

**Comment by squid-f** [comment]
> > Hi I am not good at all at drawing. Here below, I tried to illustrate the idea I had to play with symbols and lines to replace the post-horn.
> 
> Still looks like a post horn, I thought you wanted to put something else there thinking

Yes, I did. Then, I noticed it was important for you to echo the deutsche post logo and why not. So, I came with a more modern look. Actually, in many cultures, a horn has been used to make announcements. So, it might mean more than I thought, even for people leaving outside Germany.

**Comment by HerrFrutti** [comment]
Hey, if've just played around with a frew Sketches:
![Schildichat](https://user-images.githubusercontent.com/6975702/149625344-9f7e9b61-529b-4954-98f7-1c14d3ffa01d.png)
My time right now is very limited, but I just wanted to share this idea here.
I was inspired by this:
[https://www.heraldik-wiki.de/wiki/Schildkr%C3%B6te_(Wappentier)](https://www.heraldik-wiki.de/wiki/Schildkr%C3%B6te_(Wappentier))
[https://de.wikipedia.org/wiki/Westerhorn#/media/Datei:Westerhorn_Wappen.png](https://de.wikipedia.org/wiki/Westerhorn#/media/Datei:Westerhorn_Wappen.png)
Of course right now it's very much the same, but if you like the over all idea, we can change it up a bit, give it color, etc

let me know your stands :)

**Comment by SpiritCroc** [comment]
Too much detail.
Maybe as some sanity check: if it has more detail then the current icon, it is too much. The turtle in the current icon originally had more shell segments then it has now, but we had to reduce it so it looks fine as icon.

**Comment by HerrFrutti** [comment]
But the overall placement of the objects cool, or are there any suggestions ?

**Comment by SpiritCroc** [comment]
I still prefer this one:

> ![sketch1637608210896](https://user-images.githubusercontent.com/6975702/142921489-43c0ce1c-3869-4f66-916f-2094e80ae1e2.png)

**Comment by HerrFrutti** [comment]
So new try. I'm very unsure about the coloring...
![Schildichat#2](https://user-images.githubusercontent.com/6975702/149632731-072a7f28-7bef-464a-8183-4937c61e44d2.png)

**Comment by HerrFrutti** [comment]
Did some coloring:
![path1205](https://user-images.githubusercontent.com/6975702/149653353-814079ce-fe26-471b-8b72-c3e8ca6fea59.png)
[Schildichat_draft.zip](https://github.com/SchildiChat/SchildiChat-android/files/7876620/Schildichat_draft.zip)

**Comment by su-ex** [comment]
Thanks for working on this! Not ... too bad, because it's closer to your initial draft, but we still prefer the shape of your original draft. Something looks off now.

For coloring: The new icon should only use some of or the same colors as our old icon:
- The light green (our main accent color): #8bc34a
- The dark green
- The yellow
- The brighter yellow as an outline

The icon _should not_ contain any black or white, especially at the border, because this is almost always hard to distinguish from either the light or the dark background.

See this as an example: https://github.com/SchildiChat/schildichat-graphics/blob/master/graphics-web/store_icon.svg

I personally don't like the shield, especially the tip at the top.

**Comment by HerrFrutti** [comment]
okay, so the shield "format" has to be changed. How do you like the horn and the turtle head/feet etc?

**Comment by SpiritCroc** [comment]
I did prefer the pointy head and feet in your initial sketch. Horn seems okay to me, maybe too much detail, would need to check on the device.

**Comment by HerrFrutti** [comment]
ahh, so the feet and head not round, instead sharper edges, I see.
I thaught about a megaphone instead of a horn, but I think i could simplify the horn.

**Comment by su-ex** [comment]
A proper post horn should have the looping: https://en.m.wikipedia.org/wiki/Post_horn
More examples: https://de.m.wikipedia.org/wiki/Posthorn#Im_Wandel_der_Zeit

**Comment by HerrFrutti** [comment]
I'd like something like this:
https://de.m.wikipedia.org/wiki/Posthorn#/media/Datei%3APosten-norge-2008.svg

**Comment by su-ex** [comment]
Yes, such a simplified form is perfectly fine (the one from "Schwedische Post" goes in the same direction.
We should just make sure it's not too close to an existing logo.

**Comment by andrew712-1** [comment]
> Yes, such a simplified form is perfectly fine (the one from "Schwedische Post" goes in the same direction). We should just make sure it's not too close to an existing logo.

And maybe add a few dots in the middle of the horn to make it even clearer that it's a chatting app.

**Comment by su-ex** [comment]
No, that would be weird. Rather indicate the [tassles](https://en.m.wikipedia.org/wiki/Tassel).

**Comment by HerrFrutti** [comment]
This shield + turtle better?
![rect87055](https://user-images.githubusercontent.com/6975702/149662626-d38e47eb-e065-426c-90ac-64dbad3954d7.png)

**Comment by su-ex** [comment]
This looks weird.

> Hey, if've just played around with a frew Sketches: ![Schildichat_1](https://user-images.githubusercontent.com/6975702/149625575-3f74790a-04e4-4b4f-a35a-7aef407247c2.png)

Rather only the turtle from this without all the details on the shell and instead the post horn on the shell.
So a merger between this and your initial draft.

**Comment by HerrFrutti** [comment]
okay, I'll post more drafts, in a few days/weeks. But for this weekend I'm done...

Personally I like this turle the most, so I like your idea.

**Comment by su-ex** [comment]
Just keep in mind we both (@SpiritCroc and me) like your initial draft the most. My only concern about it is, that it needs to be quite high to actually look good, which might be too high for an icon.

**Comment by HerrFrutti** [comment]
Yes thats why I asked what you think about the draft:
> This shield + turtle better? ![rect87055](https://user-images.githubusercontent.com/6975702/149662626-d38e47eb-e065-426c-90ac-64dbad3954d7.png)
Because its aspect ratio is more for icons.

**Comment by su-ex** [comment]
And again: This one looks weird.
It's definitely just too short in height.
But the head is also still far too rounded compared to the initial draft.
The shape of the legs is better as well on the initial draft.

**Comment by dzg** [comment]
What about borrowing the abstract look of the Element logo ...

![](https://user-images.githubusercontent.com/466557/150048946-ec60d03a-2efb-47d6-a708-7c9a66b117be.png)

With extremities if you like ...

![](https://user-images.githubusercontent.com/466557/150049194-57f22a1c-348e-4282-8e48-05cc769d6c6d.png)

**Comment by SpiritCroc** [comment]
@su-ex and I are so disturbed by your color choice, that we can't realistically judge your proposal yet.

Some first thoughts: we both like the 2nd option of your proposals the most. While it looks modern and has the benefit of not having too much detail and having the correct icon ratio, it might be "too modern" for us, we think it looks a little boring and doesn't really stand out on the home screen. Maybe throwing in some more colors from our current logo might help. Also, @su-ex is missing the post horn ;)

**Comment by HerrFrutti** [comment]
I like this so much better than my drafts. Happy to see someone more professional gives in some thoughts. thank you!

**Comment by AnisTaluqdar** [comment]
> or more abstract ...
> <p align="center">
> <img src="https://user-images.githubusercontent.com/466557/150050492-05a4a3cf-081c-498b-a2e0-6de2d07c51bf.png" width=50%>
> </p>


Wow, it's looks cute

**Comment by dzg** [comment]
> @su-ex and I are so disturbed by your color choice, that we can't realistically judge your proposal yet.
> 
> Some first thoughts: we both like the 2nd option of your proposals the most. While it looks modern and has the benefit of not having too much detail and having the correct icon ratio, it might be "too modern" for us, we think it looks a little boring and doesn't really stand out on the home screen. Maybe throwing in some more colors from our current logo might help. Also, @su-ex is missing the post horn ;)

Color "choice" wasn't really a choice, I'm just using the Element colors

<p align=center><img src='https://user-images.githubusercontent.com/466557/150388344-21f54375-a0c9-41a1-bd0a-f27caae811cb.png' width=50% /></p>

I thought that since this is a fork of Element, it makes sense to "pay visual homage" to it.

If you look at most modern messaging apps
![image](https://user-images.githubusercontent.com/466557/150389623-4e92a8d3-ce27-4477-a18f-ef4ad72eb743.png)
they all look modern and simple to me.

Could add shading or your current color
![image](https://user-images.githubusercontent.com/466557/150394221-3b4dbd22-6992-4d8a-a194-2e10e2d043aa.png)

Not feeling the horn, sorry.

**Comment by HerrFrutti** [comment]
I like the lower left color.

A thaught about the horn:
What if you expand the turtle to the edge and put the horn inside thw middle. Expanding the turtle would give you some space.
We would loose the dots though.

**Comment by su-ex** [comment]
> pay visual homage to Element

Well, we do not like Element's style, that's one of the reasons SchildiChat exists.
Furthermore, there's no need to stress that SchildiChat is "just a fork", it should feel like a complete matrix messenger, independent of the implementation, also for users who never even tried Element.
Apart from that, SchildiChat and Element should be easily distinguishable if they are both installed (for testing or whatever reason).

Yes, other modern messaging app have very simplistic icon styles.
There's nothing wrong with that.
However, it's still nice if the icon somehow catches the user's eye (in a nice way), instead of looking just like an icon to some Android system setting by being too plain, if you understand what we mean.
Your latest suggestions are far too abstract, they bear no visual representation of our app at all.

Regarding color choice, we were thinking more about something like this:
![color choice](https://user-images.githubusercontent.com/9221456/150398898-276607cd-e1de-4d24-a401-76ad75336019.png)

**Comment by dzg** [comment]
![](https://user-images.githubusercontent.com/466557/150424806-9d015900-37af-4674-81af-57062ba8c58e.png)

**Comment by su-ex** [comment]
Still far too generic imo:
Seeing the icon and thinking of a turtle is a must. At least we're often asked what our current icon should represent. With such a generic icon, this won't change at all, rather the opposit.

Aaaaaand there needs to happen a lot so I can give up the post horn. All current suggestions don't satisfy me at all and I strongly prefer our current icon. 📯

**Comment by dzg** [comment]
![](https://user-images.githubusercontent.com/466557/150442082-e603fe18-d1f5-4a36-b595-586367fcbe36.png)

**Comment by dzg** [comment]
![](https://user-images.githubusercontent.com/466557/150442082-e603fe18-d1f5-4a36-b595-586367fcbe36.png)

**Comment by su-ex** [comment]
Not bad, but I fear it'll be far too much detail in small.
Now I'd have expected a more abstract form of a post horn like mentioned above. 😆

**Comment by dzg** [comment]
Heh, I was actually joking with that. (Totally not my style.)

**Comment by su-ex** [comment]
Haha, ok, tastes sure are different. 🎨
I think I'd prefer the head more up and not so much bent down, but apart from that, this should go straight into our sticker pack. 🤩

**Comment by SpiritCroc** [comment]
@dzg are you fine if we include your joke in our stickerpack? That would actually be a fun addition (although probably best with transparent background) :grin:

**Comment by franksouza183** [comment]
Maybe like this? Just a tweaked design and some lighting
![schildichat-desktop](https://user-images.githubusercontent.com/6532907/157344250-95c2d515-9e43-4b05-9db6-4c9fb02675c1.png)

**Comment by franksouza183** [comment]
... or like this
![schildichat-desktop-2](https://user-images.githubusercontent.com/6532907/157357456-9615aa5e-4e6b-41a8-b69f-5f50c128fa00.png)

**Comment by SpiritCroc** [comment]
These look great, I'd love to use them some way or another!
However, the 3D style is a bit too much for use as an icon (a little too much detail, and looks out of place compared to other icons).
I attached some screenshots how they compare to other icons on my Android, it's in the middle row the 2nd from the right:
![device-2022-03-09-092931](https://user-images.githubusercontent.com/12481742/157402648-1a43d815-d042-4a59-a7bc-3931fe2c5d0a.png)
![device-2022-03-09-092841](https://user-images.githubusercontent.com/12481742/157402663-59690721-d95c-445a-9162-2d96ff2614a0.png)

**Comment by su-ex** [comment]
I'm not entirely convinced by the post horn as head of the turtle ...

**Comment by SpiritCroc** [comment]
The background content is fully chosen by us, while the shape is enforced by the operating system (keyword: adaptive icons). So we can set it to anything that's not transparent. For demonstration purposes, I didn't set it, though, so my launcher made it white. For our current icon, we're using #e2f0d2, which I'd probably prefer over your choice of background green, in order to get more contrast.

The feet look a little weird right now, as if the turtle was not standing on the ground, but instead rolling around :thinking:

**Comment by franksouza183** [comment]
I bent the feets a little to follow a more circular shape for the icon as a whole.

![shildichat-desktop-3](https://user-images.githubusercontent.com/6532907/157474102-0841d1b3-816a-463f-92a7-8891a01ad2cf.png)

And a more symmetrical version:

![shildichat-desktop-10](https://user-images.githubusercontent.com/6532907/157474259-c4ece43b-0866-458c-8b0f-57d59d3d9850.png)
![shildichat-desktop-9](https://user-images.githubusercontent.com/6532907/157474262-6116c344-c43d-4335-9855-ab1390389e65.png)
![shildichat-desktop-8](https://user-images.githubusercontent.com/6532907/157474263-fa64d796-05ef-40e2-90de-cfa2fd02ad17.png)
![shildichat-desktop-7](https://user-images.githubusercontent.com/6532907/157474265-ac03b73a-b3c1-439a-9cf4-18bdbc18f5d9.png)
![shildichat-desktop-6](https://user-images.githubusercontent.com/6532907/157474268-5f1c041a-6180-4116-a2df-f7fcd37a3bda.png)

**Comment by SpiritCroc** [comment]
All icons starting with "And a more symmetrical version:" seem to be too abstract to me. And the one before has lost too much shades on the shield I think, looks too flat to me :thinking: 
But maybe also wait for @su-ex' opinion before spending more time, we both need to be happy with it. I'm also not fully sure if this is what makes users currently unhappy with the icon more happy.

**Comment by franksouza183** [comment]
one more try, base on @HerrFrutti 's sketch:
![shildichat-desktop-11](https://user-images.githubusercontent.com/6532907/157518611-047ef319-415c-4ac8-84b1-4f5f653360fc.png)

about the lack of shading in the other proposal, here:
![shildichat-desktop-5](https://user-images.githubusercontent.com/6532907/157518999-54964a97-55ac-4f6b-8323-6e6937637641.png)
![shildichat-desktop-4](https://user-images.githubusercontent.com/6532907/157519005-1afc0cdd-4d76-4275-9b54-cde2f7c656f1.png)
![shildichat-desktop-3](https://user-images.githubusercontent.com/6532907/157519008-dbf8e5d4-a74b-44f3-9417-2339e02dd142.png)

**Comment by su-ex** [comment]
Your first new draft heads in the right direction. But I think it still stands true what has been stated somewhere above:
> I did prefer the pointy head and feet in your initial sketch.

Also, the layering of the post horn feels somehow off. I'd still like to see a draft that sort of integrates the post horn into the backplate of the turtle's shell.
